main.go

```diff
@@ -432,3 +431,19 @@ func setStatus(s *discordgo.Session) {
 	update := discordgo.UpdateStatusData{Game: &game}
 	s.UpdateStatusComplex(update)
 }
+
+func rolesStuff(s *discordgo.Session, g *discordgo.GuildCreate) {
```

Comment:
This won't work, it needs to be
```suggestion
func rolesStuff(s *discordgo.Session, g *discordgo.Ready) {
```

also rename the function to initializeRoles

Reply:
Ready doesn't work, as it gets called before the guilds are fetched.

GuildCreate gets called everytime the bot starts however.

Reply:
now it works :eye::lips::eye: